### DISCUSSION TITLE
Deploy Static Site From Local Path

### DISCUSSION THREAD

**Discussion by raniaamina** [coollabsio/coolify#1436] [Feature Requests]
Hi all, is there any way to deploy static site (plain html + css + js) but the source is not from git. So what i mean is the static file is already in local server path, so only need to point coolify to that path or upload zip of static file into coolify and let it do the magic as usual :")

Hope you get my question point, thanks.

**Comment by maietta** [comment]
Bump! I'm looking for the same!

This is the reason I didn't adopt Coolify on my brand new dedicated server.

**Comment by andrasbacsai** [comment]
This will be a thing soon. I added a fully static build pack last week, which was a prerequisite for this.

**Reply by maietta** [reply]
Thank you!

**Reply by raniaamina** [reply]
Big thanks!

**Reply by ahmedbilal205** [reply]
2025 still no option to upload static folders

**Comment by pnearing** [comment]
Is this available yet?

**Comment by mr-scrpt** [comment]
Not implemented yet?

**Comment by TriHydera** [comment]
I like this part on Netlify

**Comment by n3me5is-git** [comment]
Any news on this? I'm evaluating alternatives like Dokploy

**Comment by yogan** [comment]
Is there any way to do this now? Seems like a very basic feature - just spin up a webserver and serve files from a directory.

**Reply by yogan** [reply]
I did some research and found a solution: using this minimal Docker image: https://hub.docker.com/r/flashspys/nginx-static/
Configure an app with that image and add a directory mount from a directory on your server to `/static` in the container under "Persistant Storage". Everything from there will be served by the nginx container then.
